### PR TITLE
Replay file UX

### PR DESCRIPTION
**Pull request by avanikp**
# Type of change

Replay file UX

- [ ] Bug fix
- [X] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/pages/simulation/views/simulationForm.js [hunk 1]
```diff
@@ -256,13 +270,20 @@ class SimulationForm extends LinkedComponent {
...
     );
+    */
   };
 
   addDeviceModel = () => this.deviceModelsLink.set([ ...this.deviceModelsLink.value, newDeviceModel() ]);
 
   deleteDeviceModel = (index) =>
     () => this.deviceModelsLink.set(this.deviceModelsLink.value.filter((_, idx) => index !== idx));
 
+  uploadFile = e => {
```
Comment (spryor) [inline src/components/pages/simulation/views/simulationForm.js]:
```js
Obsevable.fromPromise(UploadCall())
	.susbcribe(
		response => this.setState({ uploadedFileId: response.fileId }),
		error => console.log('Handle errpr - show top user')
	);
```

File: src/app.config.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 const Config = {
   // APIs
-  simulationApiUrl: '/devicesimulation-svc/v1/',
+  simulationApiUrl: 'http://localhost:9003/v1/',
```
Comment (saixiaohui) [inline src/app.config.js]: Change it back

File: src/components/pages/simulation/views/simulationForm.js [hunk 3]
```diff
@@ -241,9 +275,13 @@ class SimulationForm extends LinkedComponent {
       }],
       deviceModels,
       ...simulationDuration,
-      devicesDeletionRequired
+      devicesDeletionRequired,
+      replayFileId,
+      ...replayIndefinitely
     };
 
+    console.log('modelUpdates', modelUpdates);
```
Comment (saixiaohui) [inline src/components/pages/simulation/views/simulationForm.js]: Probably should remove this

File: src/app.config.js (hunk 2 above)
Comment (avanikp) [reply src/app.config.js]: reverted

File: src/components/pages/simulation/views/simulationForm.js (hunk 3 above)
Comment (avanikp) [reply src/components/pages/simulation/views/simulationForm.js]: removed

**Pull request merged**